packages/db/src/schema/conviction-claims/defs.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  bigint,
+  index,
+  integer,
+  numeric,
+  pgTable,
+  text,
+  timestamp,
+  uniqueIndex,
+  uuid,
+} from "drizzle-orm/pg-core";
+
+import { bytea } from "../custom-types.js";
+
+/**
+ * Conviction Claims table
+ *
+ * Stores conviction claim data from the conviction claims contract.
+ * Data is decoded from transaction input to capture stake duration.
+ * Each row represents a single claim where a user claimed tokens with a specific stake duration.
+ *
+ * Indexed by:
+ * - account (for user queries)
+ * - season (for filtering by season)
+ * - blockNumber (for chronological ordering)
+ * - transactionHash for uniqueness
+ */
+export const convictionClaims = pgTable(
+  "conviction_claims",
+  {
+    // Primary key
+    id: uuid("id").primaryKey().defaultRandom(),
+
+    // Claim data
+    account: text("account").notNull(), // Address that claimed
+    eligibleAmount: numeric("eligible_amount", {
+      precision: 78,
+      scale: 0,
+      mode: "bigint",
+    }).notNull(),
+    claimedAmount: numeric("claimed_amount", {
+      precision: 78,
+      scale: 0,
+      mode: "bigint",
+    }).notNull(),
+    season: integer("season").notNull(), // Season number (0, 1, 2, etc.)
+    duration: bigint("duration", { mode: "bigint" }).notNull(), // Stake duration in seconds
+
+    // Blockchain metadata
+    blockNumber: bigint("block_number", { mode: "bigint" }).notNull(),
+    blockTimestamp: timestamp("block_timestamp", {
+      withTimezone: false,
```

Comment:
Did some learning about this option... I believe we do want to set `withTimezone: true`. We can discuss more, but it came down to this summary:

> Key Takeaway

> timestamptz: Timezone-aware. Same moment in time, displayed differently based on session timezone
timestamp: Timezone-naive. Always shows the same wall-clock time regardless of timezone

> This is why timestamptz is generally recommended for storing actual moments in time (like when an event occurred), while timestamp is better for timezone-independent values (like "store opens at 9:00 AM" which should be 9:00 regardless of timezone).

Reply:
I made the change to `withTimezone: true`.  Good call there.  As far as the TIMESTAMP vs. TIMESTAMPZ,  we seem to have a mixture of timestamp usages. Do we want another PR to convert all of our schema to TIMESTAMPZ?